### PR TITLE
 add default value to the auth provider

### PR DESCRIPTION
**Pull request by jonkiky**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/pages/accessManagment/login/view.js [hunk 1]
```diff
@@ -25,7 +25,10 @@ function getRedirectPath(query) {
 
 function loginView({ classes }) {
   const { signInWithGoogle, signInWithNIH } = useAuth();
-  const { authProviders } = globalData;
+  let { authProviders } = globalData;
```
Comment (iksheth) [inline src/pages/accessManagment/login/view.js:28]: Looks like empty authProviders situation is handled below for this page.

Comment (iksheth) [reply src/pages/accessManagment/login/view.js:28]: Line 54.

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/pages/accessManagment/login/view.js (hunk 1 above)
Comment (jonkiky) [reply src/pages/accessManagment/login/view.js:28]: Yes, that's duplicate will fix it soon.

File: src/components/Auth/loginComponent.js [hunk 2]
```diff
@@ -40,6 +40,8 @@ const IndexPage = ({ classes }) => {
 
   const redirectToLogin = () => window.location.replace(`/#${loginRoute}`);
 
+  globalData.authProviders = globalData.authProviders.concat(['google']);
```
[the remark this answers is not in the document]
Comment (iksheth) [reply src/components/Auth/loginComponent.js:43]: Got it. Thanks @jonkiky

**Review by iksheth — APPROVED** [review]

**Pull request merged**